event_listener/dfusion_db/tests/models_test.py

```diff
@@ -167,11 +168,21 @@ def test_to_dictionary(self) -> None:
 
 class AuctionResultsTest(unittest.TestCase):
 
+    def test_naive_retreth_results(self):
```

Comment:
This test kind of comes out of nowhere (as in I don't understand how it relates to the PR). I'm not entirely sure this is a valuable test given that the input string would be really hard to adjust by hand if anything in the encoding changed. We's likely just copy a new instance from the e2e test.

On the other hand I understand that it would help to catch the failing e2e faster by just running unit tests.

Anyway, I'd prefer it this could be a separate PR where we can discuss this.

Reply:
Agree, included a new branch `retreath_listener_test` with only this test.